### PR TITLE
Additional fixes to remove need for Internet connection at module startup

### PR DESCRIPTION
**Pull request by ssmusoke**

### CODE REVIEW

File: omod/src/main/resources/webModuleApplicationContext.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:p="http://www.springframework.org/schema/p"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:jee="http://www.springframework.org/schema/jee"
+       xmlns:tx="http://www.springframework.org/schema/tx"
+       xmlns:aop="http://www.springframework.org/schema/aop"
+       xmlns:util="http://www.springframework.org/schema/util"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+  		    http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
```
Comment (dkayiwa) [inline omod/src/main/resources/webModuleApplicationContext.xml]: Is this file also added to fix internet connection problems? 😊

Comment (ssmusoke) [reply omod/src/main/resources/webModuleApplicationContext.xml]: @dkayiwa From my tests yes - I am not sure now what the interactions are between the different configuration files

Comment (dkayiwa) [reply omod/src/main/resources/webModuleApplicationContext.xml]: I do not believe this at all, that without having this file a module would require an internet connection. I would have to first prove it. 😊 
Can i look at the stack trace when this file is not included?

**Comment by ssmusoke** [comment]
@dkayiwa Found that I did not need the file so I have deleted it

**Pull request merged**